.env_sample

```diff
@@ -6,3 +6,8 @@ export FOREX_SERVER_TOKEN='rake secret'
 export FOREX_DATABASE_PASSWORD='Your-secret-db-password'
 export DEVISE_SECRET='rake secret'
 export SKYLIGHT_AUTHENTICATION='skylight.io'
+HOST="domain.pl"
```

Comment:
Use the keyword `export` like above. This allow a user on server to `source .env`
In our case we store the environment variable in file .bashrc. Please do this and `source .bashrc`